### DISCUSSION TITLE
Java Version Support - JDK 8 and beyond

### DISCUSSION THREAD

**Discussion by aaron-kumar** [OxalisCommunity/Oxalis-AS4#160] [Ideas]
Some Oxalis users think that Oxalis is supporting very old Java version.  
Comment - "....That is a pretty old version and experience is that some dependency updates are blocking us from updating to new Java versions. Something should be done in order to get us to new Java versions. Not sure what exactly but we can continue on this discussion and try something out....."

**Request to Oxalis Community Member**: If you see this as an issue, then please vote (thumbs up) for this issue.

**Comment by teedjay** [comment]
Next LTS Java version is just 3 months away (Java 17 LTS releases in September), so I think we could move to require at least Java 11+
Then we have crossed the "module" gap introduced by Java 9 which could introduce some issues that needs to be solved.
